features/guidance/turn-angles.feature

```diff
@@ -768,7 +768,31 @@ Feature: Simple Turns
 
                          .             .
 
-                        g             h
+                       .             .
+
+                      .             .
+
+                     .             .
+
+                    .             .
+
+                   .             .
+
+                  .             .
+
+                 .             .
+
+                .             .
+
+               .             .
+
+              .             .
+
+             .             .
+
+            .             .
+
+            g             h
```

Comment:
Also necessary to adjust to the new detectors. Since parallel roads require a certain length (here the grid size is really small), I had to adjust the length of the roads to still have the same test (here with a merged road going to the bottom).